ql/src/test/queries/clientpositive/vector_coalesce.q

```diff
@@ -75,3 +75,56 @@ SELECT cbigint, ctinyint, coalesce(cbigint, ctinyint) as c
 FROM alltypesorc
 WHERE cbigint IS NULL
 LIMIT 10;
+
+create database test;
```

Comment:
could this test case be further simplified to keep focused on the root cause?
1. is the test failing without the patch?
2. couple of things can be removed as long as the issue is reproduced (please clarify any of these below needed in order to reproduce)
a) separate 'test' database
b) explicit bucketing INTO 64 BUCKETS
c) TBLPROPERTIES: what about removing all of them? I'm pretty sure that orc code is independent of this issue
d) what about using MANAGED tables?